tests/PHPStan/Levels/data/arrayOffsetAccess-3.json

```diff
@@ -3,10 +3,5 @@
         "message": "Invalid array key type DateTimeImmutable.",
         "line": 17,
         "ignorable": true
-    },
-    {
-        "message": "Offset DateTimeImmutable does not exist on array.",
```

Comment:
As expected, I reduce the duplicates from https://github.com/phpstan/phpstan-src/pull/4387